internal/repository/base.go

```diff
@@ -220,30 +267,24 @@ func (pg *Database) InsertReport(
 	return nil
 }
 
-func (pg *Database) InsertClaim(
+func (pg *Database) InsertSnapshot(
 	ctx context.Context,
-	claim *Claim,
+	snapshot *Snapshot,
```

Comment:
This is a good opportunity to add `RETURNING` to this method, since it's basically brand new